src/main/java/seedu/address/logic/commands/AddGroupMemberCommand.java

```diff
@@ -65,11 +64,11 @@ public CommandResult execute(Model model) throws CommandException {
         try {
             groupToAddPerson = groupList.get(0);
         } catch (IndexOutOfBoundsException e) {
-            throw new CommandException(MESSAGE_INVALID_GROUP);
+            throw new CommandException(String.format(MESSAGE_INVALID_GROUP, this.personGroup));
         }
 
         if (groupToAddPerson.contains(personToGroup)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON_IN_GROUP);
+            throw new CommandException(String.format(MESSAGE_DUPLICATE_PERSON_IN_GROUP, this.name, this.personGroup));
```

Comment:
good that the person's name and group are included in the description of the exception thrown